x-pack/plugins/ml/public/application/datavisualizer/file_based/components/results_links/results_links.tsx

```diff
@@ -19,6 +19,7 @@ import {
   DISCOVER_APP_URL_GENERATOR,
   DiscoverUrlGeneratorState,
 } from '../../../../../../../../../src/plugins/discover/public';
+import { getApplication } from '../../../../util/dependency_cache';
```

Comment:
we're moving away from using this dependancy cache. 
This could be changed to:
```
const {
    services: {
      application: { navigateToUrl },
    },
  } = useMlKibana();
```